src/legends/ramp.js

```diff
@@ -117,6 +121,7 @@ export function legendRamp(color, options) {
 
     svg
       .append("g")
+      .attr("opacity", opacity)
```

Comment:
Implement as fill-opacity for performance?

```suggestion
      .attr("fill-opacity", opacity)
```